interfaces/README.md

```diff
@@ -72,6 +74,30 @@ Following the configuration of the 121 Demo environment, the following CSP is se
   - When SSO using Azure Entra will be used, `https://login.microsoftonline.com` should be included..
 - `frame-ancestors`:  
   Allows an interface to be included in an `<iframe>` on some other web-site or -service.
-  - When the integration with Redline/Twilio Flex will be used, `https://flex.twilio.com` should be included.
 
-For each different interface, on each specific instance, other settings apply, depending on their (required) functionality.
+  When the integration with Twilio Flex will be used, `https://flex.twilio.com` should be included.
+
+  ##### Testing Twilio Flex integration
+
+  Some aspects of the Twilio Flex integration can be simulated/mocked (locally) by injecting an `<iframe>` into a page on the Twilio Flex website.
+
+  1. Go to <https://flex.twilio.com/>
+  2. Open the browser's developer tools (F12 or ⌘+⇧+I) or console. Paste and run the following code:
```

Comment:
```suggestion
  2. Without logging in, open the browser's developer tools (F12 or ⌘+⇧+I) or console. Paste and run the following code:
```

Reply:
Ah. Good point.
I haven't logged-in recently, so to me it was obvious this way. But I'd guess it wouldn't work when logged-in indeed.

```suggestion
  2. Without being logged in, open the browser's developer tools (F12 or ⌘+⇧+I) or console. Paste and run the following code:
```